consensus/src/chained_bft/chained_bft_consensus_provider.rs

```diff
@@ -31,18 +30,20 @@ use std::{convert::TryFrom, sync::Arc};
 use tokio::runtime;
 use vm_runtime::MoveVM;
 
-struct InitialSetup {
-    author: Author,
-    signer: ValidatorSigner,
-    validator: ValidatorVerifier,
+pub struct InitialSetup {
```

Comment:
Let's document this struct with something (maybe better than this)

/// The state necessary to begin state machine replication considering the local persistent storage (i.e. storage, consensus and executor state).